browser/css/jssidebar.css

```diff
@@ -353,6 +353,14 @@ button#button2.ui-pushbutton.jsdialog.sidebar {
 	margin-block-end: 8px;
 }
 
+#NavigatorPanel .ui-treeview-entry.highlighted > *:not(.ui-treeview-expander-column) {
+	background-color: var(--color-treeview-highlight);
+}
+
+.ui-treeview-expandable.highlighted .ui-treeview-cell-text {
+	color: rgb(var(--doc-type)) !important;
```

Comment:
@pedropintosilva I notice in dark mode that this isn't very visible:
<img width="264" height="230" alt="Pasted image 20250808141858" src="https://github.com/user-attachments/assets/200fb0ba-cfcf-44f8-847e-8b76d5e0a0fd" />